oauth2l/__init__.py

```diff
@@ -222,14 +225,21 @@ def _GetApplicationDefaultCredentials(scopes):
             return credentials.create_scoped(scopes)
 
 
+def _GetCredentialsFilename(credentials_filename):
+    return os.path.expanduser(credentials_filename or '~/.oauth2l.token')
+
+
+def _GetCredentialStore(credentials_filename, key_id, scopes):
+    credentials_filename = _GetCredentialsFilename(credentials_filename)
+    storage_key = '{}-{}'.format(key_id, scopes)
```

Comment:
Actually I took the "-" from this example https://github.com/google/oauth2client/blob/master/oauth2client/contrib/multiprocess_file_storage.py#L60